notebooks/ch16/classification_cost.md

```diff
@@ -195,4 +205,55 @@ The surprisingly simple gradient expression allows us to fit a logistic model to
 
 ## Summary
 
-The cross entropy cost that we use to fit the logistic model. Since it is a convex function, we use gradient descent to fit the model to the cost. We now have the necessary components of logistic regression: the model, cost function, and minimization procedure.
+The cost function for logistic models is based on cross-entropy loss. Since it is a convex function, we use gradient descent to fit the model to the cost. We now have the necessary components of logistic regression: the model, cost function, and minimization procedure.
+
+## KL Divergence and Cross-Entropy Cost
+
+### Defining KL Divergence
+
+As noted at the beginning of the section, the goal in our binary classification example is to closely approximate the real data distribution using a logistic model with parameters $\theta$:
```

Comment:
I agree, especially since we are making this a new section it will be helpful to provide more context. Originally I wasn't sure how much detail to add in order to prevent it from becoming too long, but this definitely works for the new section.